docs/config/lua/config/default_cwd.md

```diff
@@ -0,0 +1,47 @@
+# `default_cwd`
+
+Sets the default current working directory used by the initial window. If
+`wezterm start --cwd /some/path` is used to specify the current working
+directory, that will take precedence.
+
+Commands launched using [`SpawnCommand`](../SpawnCommand.md) will use the
+`cwd` specified in the `SpawnCommand`, if any.
+
+Panes/Tabs/Windows created after the first will generally try to resolve the
+current working directory of the current Pane, preferring
+[a value set by OSC 7](../../../shell-integration.markdown) and falling back to
+attempting to lookup the `cwd` of the current process group leader attached to a
+local Pane. If no `cwd` can be resolved, then the `default_cwd` will be used.
+If `default_cwd` is not specified, then the home directory of the user will be
+used.
+
+## Example Flowchart
+
+```text
+                             Is initial window?
+               ______________________|______________________
+              |                                             |
+             Yes                                            No
+              |                                             |
+       Opened with CLI                  Opened with a `SpawnCommand` that includes
+      and `--cwd` flag?                  `cwd` or is a new pane, tab, or window?
+    __________|__________                         __________|__________
+   |                     |                       |                     |
+  Yes                    No                      No                   Yes
+   |                     |                       |                     |
+  Use                    |                Is there a value    Use `cwd` specified
+`--cwd`                  |                 set by OSC 7?       by `SpawnCommand`
+                         |             __________|__________
+                         |            |                     |
+                         |            No                   Yes
+                         |____________|                     |
+                               |                        Use OSC 7
+                        Is `default_cwd`                  value
+                            defined?
+                     __________|__________
+                    |                     |
+                   Yes                    No
+                    |                     |
+                   Use                 Use home
+              `default_cwd`            directory
+```
```

Comment:
I am not sure if this is too verbose for you or not. I felt it rendered pretty well:

![Screenshot from 2021-01-25 16-58-36](https://user-images.githubusercontent.com/2488926/105786737-8045e580-5f32-11eb-9e2b-35316f24951b.png)